src/db/vco_effects.loc.tsv

```diff
@@ -18,6 +18,7 @@ effects_description_vco_ogr_greasus_speed	Speed: %+n% for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_charge_bonus	Charge Bonus: %+n for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_armour	Armour: %+n for Greasus Goldtooth	true
 effects_description_vco_ogr_greasus_ward_save	Ward save: %+n% for Greasus Goldtooth	true
+effects_description_vco_ksl_ort_flagellant_unlock_dummy	Unlocks Flagellants	true
```

Comment:
I would change the text to "Unlock new units: Flagellants". We also have to take into account these units are DLC limited, do we really want to use them?

Reply:
Probably not. Let's find another unit, or replace the objective with something else.

Reply:
I think the Luminark of Hysh still has a "religious" theme and has the benefit of not being a  DLC unit.

Reply:
A good idea as any, I'll look into adding it

Reply:
Looks done. Effect added to payload:
![image](https://user-images.githubusercontent.com/46346590/192100704-39da17ae-1436-4aa1-b0ae-19f281bcebc5.png)

Unit added to Harbour (they'd get it from the empire by trade, shipped by sea):
![image](https://user-images.githubusercontent.com/46346590/192100730-71c6afa4-9994-4b72-991d-596eab7cfc13.png)

Added function to lock/unlock, and listeners to trigger them